pkg/comp-functions/functions/vshnminio/utils.go

```diff
@@ -0,0 +1,14 @@
+package vshnminio
+
+import (
+	vshnv1 "github.com/vshn/appcat/v4/apis/vshn/v1"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+)
+
+func GetPlan(comp *vshnv1.VSHNMinio, iof *runtime.Runtime) string {
```

Comment:
Just a random idea: we could make this a method of the `VSHNDBaaSSizeSpec`.

Reply:
Done